<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QWidget" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>学号</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>姓名</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>出生日期</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>专业</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>班级</string>
    </property>
   </row>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>科目</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>成绩</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学期</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学期：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
